cms/envs/aws.py

```diff
@@ -196,7 +196,9 @@
 
 # Theme overrides
 THEME_NAME = ENV_TOKENS.get('THEME_NAME', None)
-COMPREHENSIVE_THEME_DIR = path(ENV_TOKENS.get('COMPREHENSIVE_THEME_DIR', COMPREHENSIVE_THEME_DIR))
+COMPREHENSIVE_THEME_DIRS = ENV_TOKENS.get('COMPREHENSIVE_THEME_DIRS', COMPREHENSIVE_THEME_DIRS) or []
```

Comment:
I made a similar comment at https://github.com/edx/configuration/pull/3099 -- after thinking about this a bit mroe we need to support the existing COMPREHENSIVE_THEME_DIR parameter because there are a lot of Open edX operators using theming and we don't want to introduce a breaking change -- it was different with Otto because the probability of anyone using the first version of theming that we had introduced was very low.